django/utils/autoreload.py

```diff
@@ -216,14 +216,14 @@ def get_child_arguments():
     executable is reported to not have the .exe extension which can cause bugs
     on reloading.
     """
-    import django.__main__
-    django_main_path = Path(django.__main__.__file__)
+    import __main__
     py_script = Path(sys.argv[0])
 
     args = [sys.executable] + ['-W%s' % o for o in sys.warnoptions]
-    if py_script == django_main_path:
+    if __main__.__spec__ is not None and __main__.__spec__.parent:
```

Comment:
Can we add a small comment with a link to `https://docs.python.org/3/reference/import.html#main-spec` explaining this?

Reply:
If the [comment in the subsequent two lines](https://github.com/django/django/blob/b316a26ea2ab5b72fe1f696a0332a1347d7fecf2/django/utils/autoreload.py#L224-L225) is insufficient, what did you have in mind?